### ISSUE TITLE
[lucius] @supports mixing problem?

### ISSUE THREAD

**Issue by jeongoon** [yesodweb/shakespeare#255]
```css
.status::before {
    content: 'CSS Parallax is: NOT SUPPORTED';
    color: #c00;
    display: block;

}

@supports ((perspective: 1px) and (not (-webkit-overflow-scrolling: touch))) {
    .status::before {
        content: 'CSS Parallax is: SUPPORTED';
        color: #090;
    }
}
```

I'm trying to apply css from "https://keithclark.co.uk/articles/practical-css-parallax/feature-detection/"
but lucius produced something I don't want
the result looks like:

```css
.status::before {
    content: 'CSS Parallax is: NOT SUPPORTED';
    color: #ccc
    display: block;
}
color: #090;
    }
}
```

I'm trying to avoid the what happened here. or should I put some css into "static" ?

**Comment by snoyberg** [comment]
`@supports` isn't currently supported by Lucius, but a PR to add support with appropriate tests would definitely be welcome.

**Comment by jeongoon** [comment]
It has been more than 1 year. 😅 I believe that a reponsive web is must. So i tried to fix with PR[#263] (https://github.com/yesodweb/shakespeare/pull/263#issue-1153954015)

**Comment by snoyberg** [comment]
Implemented via #263, thanks!

**Comment by jeongoon** [comment]
thank you too!!
